.github/workflows/release.yml

```
@@ -0,0 +1,52 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+      - "!v*.*.*-**"
+
+jobs:
+  ensure-version:
+    runs-on: ubuntu-latest
+    name: Ensure .version file is up to date
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Ensure version
+        run: |
+          if [ ! -f ".version" ]; then
+            echo "No .version file found"
+            exit 1
+          fi
+          if [ "v$(cat .version)" != ${{ github.ref_name }} ]; then
+            echo "Version in "".version" file does not match tag pushed."
```

We can add the complexity, but we need to explain what's going on here.
```suggestion
            echo 'The release has failed.'
            echo
            echo The version in ".version" file "(v$(cat .version))" does 
            echo not match tag pushed (${{ github.ref_name }}). The releaser
            echo needs to manually delete the release and any associated git
            echo tag, update the .version file and try again.
```